plugins/optimization-detective/class-od-preload-link-collection.php

```diff
@@ -166,6 +166,37 @@ public function get_html(): string {
 		return implode( '', $link_tags );
 	}
 
+	/**
+	 * Gets the HTTP Link header string.
+	 *
+	 * @return string HTTP Link header.
+	 */
+	public function get_headers(): string {
+		$link_headers = array();
+
+		foreach ( $this->get_adjacent_deduplicated_links() as $link ) {
```

Comment:
This foreach has quite a bit of duplication with `get_html()`. Can the common code be factored out into a private helper method?

Reply:
Sure makes sense, will do that